File: Esame_28/JAVA/Veicolo.java
public class Veicolo {
	protected int codiceVeicolo;
	protected String tipoVeicolo;
	protected String targaVeicolo;

	protected String modelloVeicolo;
	protected String marcaVeicolo;

	public Veicolo(int codiceVeicolo, String tipoVeicolo, String targaVeicolo, String modelloVeicolo,
			String marcaVeicolo) {
		this.codiceVeicolo = codiceVeicolo;
		this.tipoVeicolo = tipoVeicolo;
		this.targaVeicolo = targaVeicolo;
		this.modelloVeicolo = modelloVeicolo;
		this.marcaVeicolo = marcaVeicolo;
	}
	// Metodi

	// toString punto 3
	public String toString() {
		return this.tipoVeicolo + "\t" + this.targaVeicolo + "\t" + this.codiceVeicolo + "\t" + this.modelloVeicolo
				+ "\t" + this.marcaVeicolo + "\t";
	}

	// metodo per punto 4
	public String getTipoVeicolo() {
		return this.tipoVeicolo;
	}
}

File: Esame_28/PYTHON/Veicolo.py
class Veicolo:
    # COSTRUTTORE
    def __init__(self, codiceVeicolo, tipoVeicolo, targaVeicolo, modelloVeicolo, marcaVeicolo):
        self._codiceVeicolo = codiceVeicolo
        self._tipoVeicolo = tipoVeicolo
        self._targaVeicolo = targaVeicolo
        self._modelloVeicolo = modelloVeicolo
        self._marcaVeicolo = marcaVeicolo

    # TOSTRING PUNTO 3
    def toString(self):
        return self._tipoVeicolo + "\t" + self._targaVeicolo + "\t" + str(self._codiceVeicolo) + \
               "\t" + self._modelloVeicolo + "\t" + self._marcaVeicolo

    # METODO PER PUNTO 4
    def getTipoVeicolo(self):
        return self._tipoVeicolo
